src/claude-session-utility.ts

```
@@ -1893,12 +1914,29 @@ export const loadSessionEnrichment = async (
+            const parsed = JSON.parse(lastLine(recapOutput));
+            // The line ends with a UI hint that is not part of the summary.
+            const text = String(parsed.content || '')
+              .replace(/\s*\(disable recaps in \/config\)\s*$/, '')
+              .trim();
+            if (text) {
+              recaps.set(session.sessionId, {
+                text,
+                at: typeof parsed.timestamp === 'string' ? parsed.timestamp : '',
+              });
+            }
+          } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)